nservicebus/messaging/messages-events-commands.md

```diff
@@ -44,6 +44,12 @@ Used to communicate that some action has taken place. An *Event* should be _publ
 
 Note: For reply messages in a request and response pattern, use `IMessage` since these replies are neither a Command nor an Event.
 
+### Command/Event Comparison
+
+Here is a quick side-by-side comparison of some of the differences between commands and events:
```

Comment:
The differences laid out as a table is hard to understand and will be harder still for a person that is new to these terms.  Rather than having a comparison table, these must be included as part of the command and event definition where it is much easier to read. And also gives a whole lot of more context into what a command is or what an event is.